tests/power_analysis/test_power_analysis_with_experiment_data.py

```diff
@@ -0,0 +1,55 @@
+from itertools import product
```

Comment:
can we add the following two tests?
* AA test: check that if effect = 0, power = alpha, more or less
* Check that if effect is bigger, power is bigger